src/StyleSheet.js

```diff
@@ -20,8 +18,7 @@ import RulesContainer from './RulesContainer'
  * @api public
  */
 export default class StyleSheet {
-  constructor(rules, options) {
-    const named = options.named == null ? true : options.named
+  constructor(styles, options) {
```

Comment:
For naming consistency. We call user style definitions styles, they become rules when they are wrapped into a rule class instance.